spec/sections/copies.bs

```diff
@@ -29,8 +29,16 @@ and "immediate" {{GPUQueue}} operations:
 - {{GPUQueue/copyExternalImageToTexture()}}, for copies from Web Platform image sources to textures
 
 Some texel values have <dfn dfn lt='equivalent texel representation'>multiple possible representations</dfn>
-of some values, e.g. as `r8snorm`, -1.0 can be represented as either -127 or -128.
-Copy commands are not guaranteed to preserve the source's bit-representation.
+of some values. As a result copy commands are not guaranteed to preserve the source's bit-representation.
+Examples include:
+
+ - snorm values may represent -1.0 as either -127 or -128.
+ - Any NaN or Infinity may be represented by an indeterminant, non-signaling NaN.
+ - Any subnormal value, -0.0, or +0.0 may be represented by either -0.0 or +0.0.
```

Comment:
Just some wording suggestions ("represent" isn't quite right because subnormals/nans/infinities aren't really "represented" if they get lost)
```suggestion
- snorm values may represent -1.0 as either -127 or -128.
- The signs of zero values may not be preserved.
- Subnormal floating-point values may be either preserved or replaced by either `-0.0` or `+0.0`.
- Any NaN or Infinity is an invalid value and may be replaced by an [=indeterminate value=].
```

Reply:
Did some minor rephrasing, but mostly used this suggestion.